### PR TITLE
970: Add checkbox for email usage in application form

### LINKED ISSUE #970: [Bavaria] Add social media checkbox
**Is your feature request related to a problem? Please describe.**
As a service user of a region i want an applicant to explicitly agree for communication.

*Acceptance criteria*

- add a checkbox in step 4 in the application process with the label "Meine E-Mailadresse darf im Sinne der Datenschutzerklärung für Kommunikationsmaßnahmen (z.B. Newsletter, Hinweise zu Gewinnspielen, ...) rund um das Thema Ehrenamt genutzt werden"
- checkbox is default unchecked
- send this information with the application to the backend (extend JSON Data)

### PR DESCRIPTION
**Pull request by steffenkleinle**
Fixes #970

### CODE REVIEW

**Review by f1sh1918 — APPROVED** [review]
Tested on chrome. Works as expected :)

File: administration/src/mui-modules/application/forms/StepSendForm.tsx [hunk 1]
```diff
@@ -83,6 +91,12 @@ const StepSendForm: Form<State, Options, ValidatedInput, AdditionalProps> = {
           setState={setGivenInformationIsCorrectAndComplete}
           options={givenInformationIsCorrectAndCompleteOptions}
         />
+        <SubForms.hasAcceptedEmailUsage.Component
+          label='Meine E-Mailadresse darf im Sinne der Datenschutzerklärung für Kommunikationsmaßnahmen (z.B. Newsletter, Hinweise zu Gewinnspielen, ...) rund um das Thema Ehrenamt genutzt werden.'
```
Comment (sarahsporck) [inline administration/src/mui-modules/application/forms/StepSendForm.tsx:95]: https://www.duden.de/rechtschreibung/E_Mail_Adresse
```suggestion
          label='Meine E-Mail-Adresse darf im Sinne der Datenschutzerklärung für Kommunikationsmaßnahmen (z.B. Newsletter, Hinweise zu Gewinnspielen, ...) rund um das Thema Ehrenamt genutzt werden.'
```

File: backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/schema/create/Application.kt [hunk 2]
```diff
@@ -107,7 +108,13 @@ data class Application(
                     mapOf("de" to "Ich versichere, dass alle angegebenen Informationen korrekt und vollständig sind"),
                     Type.Boolean,
                     givenInformationIsCorrectAndComplete
-                )
+                ),
+                JsonField(
+                    "hasAcceptedEmailUsage",
+                    mapOf("de" to "Meine E-Mailadresse darf im Sinne der Datenschutzerklärung für Kommunikationsmaßnahmen (z.B. Newsletter, Hinweise zu Gewinnspielen, ...) rund um das Thema Ehrenamt genutzt werden."),
```
Comment (sarahsporck) [inline backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/schema/create/Application.kt:114]:
```suggestion
                    mapOf("de" to "Meine E-Mail-Adresse darf im Sinne der Datenschutzerklärung für Kommunikationsmaßnahmen (z.B. Newsletter, Hinweise zu Gewinnspielen, ...) rund um das Thema Ehrenamt genutzt werden."),
```

**Review by sarahsporck — APPROVED** [review]
Tested in firefox.

**Pull request merged**